cmd/config/init/output.go

```diff
@@ -20,6 +28,24 @@ func printInitOutput(addresses []utils.AddressData, rollappId string) {
 			filteredAddresses = append(filteredAddresses, address)
 		}
 	}
-	utils.PrintAddresses(filteredAddresses)
-	fmt.Printf("\n🔔 Please fund these addresses to register and run the rollapp.\n")
+	return filteredAddresses
+}
+
+func FormatTokenSupplyLine(rollappConfig utils.RollappConfig) string {
+	displayDenom := strings.ToUpper(rollappConfig.Denom[1:])
+	return fmt.Sprintf("💰 Total Token Supply: %s %s. Note that 1 %s == 1 * 10^%d %s (like 1 BTC == 1 * 10^8 satoshi).",
```

Comment:
Why not giving the ETH example? we set our decimals according to that

Reply:
good idea